v2/util/docker-cleanup.sh

```diff
@@ -6,9 +6,12 @@ if [[ $(/usr/bin/docker ps -a | grep Exited | awk '{print $1 "\\t" $7 "\\t" $10}
 then
   echo "Starting docker cleanup with a max of: $MAX_DOCKER"
   /usr/bin/docker ps -a | grep Exited | awk '{print $1 "\\t" $7 "\\t" $10}' | grep week | awk '{print $1}' | head -n $MAX_DOCKER | xargs /usr/bin/docker rm -v
-  /usr/bin/docker images -a | grep none | awk '{print $3}' | xargs /usr/bin/docker rmi
-  echo "cleanup complete"
+  echo "container cleanup complete"
 else
-  echo "nothing to cleanup"
+  echo "no containers to cleanup"
 fi
+
+/usr/bin/docker images -a  | awk '{print $3}' | xargs /usr/bin/docker rmi
```

Comment:
So @cfortier2 this PR is a result of one of the nodes running out of disk space.  Upon investigation, we found nearly 40GB of docker images that were no longer used, and running the above command fixed it.

The original command also had a `| grep none` in it, which caused it to miss quite a few images.